google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/BlobSnippets.java

```diff
@@ -173,6 +173,22 @@ public Blob copyToStrings(String bucketName, String blobName) {
   }
 
   /**
+   * Example of moving a blob to a different bucket with a different name.
+   */
+  public Blob moveTo(Blob srcBlob, String destBucket, String destBlob) {
```

Comment:
> You should use the blob field instead of passing a blob parameter.

It's just problematic because moving the blob includes deleting it, which makes testing it murkier (the way the tests are structured right now). ie if I move the `blob` field while testing `moveTo`, then the test for `delete` has nothing to delete. I figured my options were:

1. Move a blob that's created via one of the `copyTo` tests instead of the canonical `blob`
1. Change the structure of the tests such that each method tested is a separate test method, and the blob is created and torn down for each test
1. Create another test that creates, uses, and cleans up its own blob instead of using the one in `beforeClass`.

Clearly I went with the first option, which seemed the simplest. The second option seems like the Right Thing to do, but would involve rewrite a bunch of the other tests, and seems like it'd incur overhead, which I don't know how much your team cares about. The third option seems like a decent compromise between keeping the actual snippets file using the same `blob` convention and keeping changes to the test file to a minimum, but then the test file has inconsistent test conventions, which feels hacky.

Thoughts?

Also - I removed the `TARGET` and `VARIABLE` tags, first because I wasn't sure they were required, but more importantly because I didn't know how to specify a `VARIABLE` for a `Blob` type :-)